File: sop-sdk/sdk-csharp/SDKCSharp/Response/GetStoryResponse.cs
using System;
using Newtonsoft.Json;

namespace SDKCSharp.Response
{
    public class GetStoryResponse: BaseResponse
    {
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("gmt_create")]
        public string GmtCreate { get; set; }

    }
}

File: sop-sdk/sdk-java/src/main/java/com/gitee/sop/sdk/response/GetStoryResponse.java
package com.gitee.sop.sdk.response;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

@Data
@EqualsAndHashCode(callSuper = false)
public class GetStoryResponse extends BaseResponse {
    private Long id;
    private String name;
    private Date gmt_create;
}
